packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -489,6 +496,10 @@ export class Function extends FunctionBase {
       managedPolicies.push(iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AWSLambdaVPCAccessExecutionRole'));
     }
 
+    if (props.filesystems) {
+      managedPolicies.push(iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonElasticFileSystemClientFullAccess'));
+    }
+
```

Comment:
@nija-at I moved the statement to here. Is it fine?

Reply:
Yes, this is correct.